content/it/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,314 @@
+è impostata su `IfNotPresent` (se non presente) se non specificato diversamente.
+Questa policy permette al
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} di evitare di fare il pull
+di un'immagine se questa è già presente.
+
+Si può forzare il pull:
+
+- impostando `imagePullPolicy` (specifica per il pull delle immagini) del container su `Always` (sempre).
+- omettendo `imagePullPolicy` ed usando il tag `:latest` (più recente) per l'immagine da utilizzare;
+  Kubernetes imposterà la policy su `Always` (sempre).
+- omettendo `imagePullPolicy` ed il tag per l'immagine da utilizzare.
+- abilitando l'admission controller [AlwaysPullImages](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages).
+
+{{< note >}}
+Il valore dell'impostazione `imagePullPolicy` del container è sempre presente quando l'oggetto viene creato per la prima volta
+e non viene aggiornato se il tag dell'immagine dovesse cambiare successivamente.
+
+Ad esempio, creando un Deployment con un'immagine il cui tag _non_ è 
+`:latest`, e successivamente aggiornando il tag di quell'immagine a `:latest`, il campo
+ `imagePullPolicy` _non_ cambierà su `Always`. 
+È necessario modificare manualmente la policy di pull di ogni oggetto dopo la sua creazione.
+{{< /note >}}
+
+Quando `imagePullPolicy` è definito senza un valore specifico, esso è impostato su `Always`.
+
+## Immagini Multi-architecture con indici
+
+Oltre a fornire immagini binarie, un _container registry_ può fornire un [indice per l'immagine di un container](https://github.com/opencontainers/image-spec/blob/master/image-index.md). 
+L'indice di un'immagine può puntare a più [file manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md) ciascuno per una versione specifica dell'architettura di un container. 
+L'idea è che si può avere un unico nome per una stessa immagine (ad esempio: `pause`, `example/mycontainer`, `kube-apiserver`) e permettere a diversi sistemi di recuperare l'immagine binaria corretta a seconda dell'architettura della macchina che la sta utilizzando.
+
+
+Kubernetes stesso tipicamente nomina le immagini dei container tramite il suffisso `-$(ARCH)`. 
+Per la garantire la retrocompatibilità è meglio generare le vecchie immagini con dei suffissi. 
+L'idea è quella di generare, ad esempio, l'immagine `pausa` che ha il manifest per tutte le versioni del processore _arch_ e
+ `pausa-amd64` che è retrocompatibile per le vecchie configurazioni o per quei file YAML in cui sono specificate le immagini con i suffissi.
+
+## Utilizzare un registro privato
+
+I registri privati possono richiedere l'utilizzo di chiavi per accedere alle immagini in essi contenute.  
+Le credenziali possono essere fornite in molti modi:
+  - configurando i nodi in modo tale da autenticarsi al registro provato
+    - tutti i pod possono acquisire informazioni da qualsiasi registro privato configurato
```

Comment:
Se non sbaglio questa frase non è presente nella versione originale
```suggestio
```

Reply:
(typo in `suggestio` @fabriziopandini )

Reply:
@fabriziopandini  I think this is the sentence: `all pods can read any configured private registries`